src/main/java/Dao/CartDao.java

```diff
@@ -0,0 +1,94 @@
+package Dao;
+
+import Entities.Cart;
+

```

Comment:
remove the extra blank line

Reply:
Done.

Reply:
I can still an extra blank line

Reply:
I have added lines for the readability purpose. These spaces indicate starting from package, then space to separate from project imports then a space to separate from java imports.